src/views/Home.jsx

```diff
@@ -10,25 +10,60 @@ export function Home({ onClick, onChange, handleForm }) {
 	}, [token, onChange]);
 
 	return (
-		<div className="Home">
-			<button onClick={onClick} className="create-list-btn">
-				Create a new list
+		<div
+			className="home mt-20 flex flex-col
+		items-center justify-center "
+		>
+			<button
+				onClick={onClick}
+				className="create-list-btn bg-[#C2410C] text-white w-[243px] lg:w-[534px] h-[46px] lg:h-[90px]
+				rounded-[10px] font-poppins font-bold text-[20px] lg:text-[32px] leading-[30px] mb-16 border border-1 border-black
+				"
```

Comment:
I think 32px for text is too big. This is what I see in the browser
![image](https://user-images.githubusercontent.com/74952593/203812116-d40d0410-3b99-4c3d-95ae-5e531ecd34b9.png)